catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/data/ShareableAttributes.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms.data;
+
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardTypeImpl;
+import ddf.catalog.data.types.Core;
+import java.util.Set;
+
+/**
+ * Set of attributes needed to enable sharing on Metacards. Should be refactored to live in a more
+ * central location at a later time.
+ *
+ * <p><i>This code is experimental. While it is functional and tested, it may change or be removed
+ * in a future version of the library.</i>
+ *
+ * <p>TODO DDF-3671 Revisit sharing functionality for metacards
+ */
+@SuppressWarnings("squid:S1135" /* Action to-do has a ticket number and will be addressed later */)
+public class ShareableAttributes extends MetacardTypeImpl {
+  public static final String SHAREABLE_TAG = "shareable";
+
+  public static final String SHAREABLE_METADATA = "metacard.sharing";
+
+  // @formatter:off
+  private static final Set<AttributeDescriptor> SHAREABLE_ATTRIBUTES =
+      ImmutableSet.of(
+          new AttributeDescriptorImpl(
+              Core.METACARD_OWNER,
+              true /* indexed */,
+              true /* stored */,
+              true /* tokenized */,
+              false /* multivalued */,
+              BasicTypes.STRING_TYPE),
+          new AttributeDescriptorImpl(
+              SHAREABLE_METADATA,
```

Comment:
❗️ This should be replaced with `SecurityAttribute`s.  That is what the workspaces use for sharing control.  

https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/SecurityAttributes.java
https://github.com/codice/ddf/blob/908a975df74a7f345eef2964cc2ccdde58e4db53/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/WorkspacePolicyExtension.java

Reply:
I can remove this, but something seems off. I think the definition is incorrect in `WorkspaceAttributes` where it's listed as `XML_TYPE` but should have been `STRING_TYPE`. 
https://github.com/codice/ddf/blob/908a975df74a7f345eef2964cc2ccdde58e4db53/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceAttributes.java#L69-L74

Note how `WorkspaceMetacardImpl` is interacting with that attribute: 
https://github.com/codice/ddf/blob/908a975df74a7f345eef2964cc2ccdde58e4db53/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardImpl.java#L172-L179

Reply:
Furthermore if these attributes should be the ones from `SecurityAttributes` then what is this `WORKSPACE_SHARING` attribute being used for?

Reply:
I don't think that attribute is used as far as I could tell.  None of the workspace security related plugins use that attribute.  Only test code seems to use it.  I think it is left over from a previous failed attempt at workspace sharing.